mythical-creatures/exercises/hobbit.js

```diff
@@ -0,0 +1,22 @@
+function Hobbit(name, disposition, age, adult, isShort, old){
+  this.name = name;
+  this.disposition = disposition || 'homebody';
+  this.age = age || 0;
+  this.adult = () => {
+    if(this.age >= 33){
```

Comment:
since you are returning `true` or `false`, which is also what your condition will return, you can get away with just using the condition. 

`this.adult = () => this.age >= 33`